kano_greeter/password_view.py

```diff
@@ -19,6 +19,8 @@
 from kano_greeter.last_user import set_last_user
 from kano_greeter.user_list import KanoUserList
 
+# Placeholder function to translate encoding only when necessary
+def encode(x): return x.encode('utf-8') if isinstance(x,unicode) else x
```

Comment:
This looks like it might be a useful function to add to the `kano_i18n` library.